usdigitalresponse/usdr-gost
# 266 hide links on dashboard 2

### Ticket #
#266
### Description
Apologies for the size of the changes. Some of them are just minor cleanups. If you want me to create a new branch and PR that don't contain those changes, let me know.
### Screenshots / Demo Video
n/a
### Testing

To manually test (using local database):
- Get local database up-to-date with current day: `docker compose exec app yarn db:seed`
- Verify dashboard links are visible
- Click on each link to verify 5 total "Recent Activity" and 4 total "Upcoming..." grants.
- Comment out as shown in the following diff:

```
--- a/packages/server/seeds/dev/ref/grants.js
+++ b/packages/server/seeds/dev/ref/grants.js
@@ -754,6 +754,7 @@ const grantsInterested = [
         updated_at: daysAhead(5),
         interested_code_id: interestedId('Will Support'),
     },
+/*
     {
         grant_id: grants[6].grant_id,
         agency_id: usdr,
@@ -762,6 +763,7 @@ const grantsInterested = [
         updated_at: daysAhead(6),
         interested_code_id: interestedId('Will Support'),
     },
+*/
```

- Re-run `docker compose exec app yarn db:seed`
- Verify dashboard links are gone

### Checklist
